File: linmour-order-module/linmour-order-api/src/main/java/com/linmour/order/pojo/Dto/CreateOrderDto.java
package com.linmour.order.pojo.Dto;

import com.linmour.order.pojo.Do.OrderItem;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.math.BigDecimal;
import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class CreateOrderDto implements Serializable {

    private String orderId;

    private Long tableId;
    private BigDecimal amount;
    private List<OrderItem> orderItems;
    private String remark;
    private Long shopId;
}

File: linmour-order-module-go/server/models/CreateOrderDto.go
package models

type CreateOrderDto struct {
	OrderId    string      `json:"orderId"`
	TableId    int64       `json:"tableId"`
	Amount     float64     `json:"amount"`
	OrderItems []OrderItem `json:"orderItems"`
	Remark     string      `json:"remark"`
	ShopId     int64       `json:"shopId"`
}
